entity-framework/core/managing-schemas/migrations/managing.md

```diff
@@ -44,6 +44,8 @@ The timestamp in the filename helps keep them ordered chronologically so you can
 
 You are free to move Migrations files and change their namespace manually. New migrations are created as siblings of the last migration. Alternatively, you can specify the namespace at generation time as follows:
 
+<!--markdownlint-disable MD024-->
```

Comment:
Is this suppression necessary given that there's only one tab set within this heading ("Namespaces")?

Reply:
There is an earlier tab set up on L19

Reply:
I think if "Namespaces" was an H2, they wouldn't be siblings, but right now, they are

Reply:
Oh that's a bug on our part - the tab headings are supposed to be part of the Namespaces section, so they're supposed to be an H4 within the Namespaces H3.

I've pushed a commit correcting this and removing the suppression, thanks!